README.md

```diff
@@ -1,12 +1,14 @@
-# SpatialOS GDK for Unreal Starter Project
+# SpatialOS GDK for Unreal Test Gyms Project 
 
 * **LICENSE:** Use of the contents of this repository is subject to the [license](LICENSE.md)
-* **REPOSITORY:** github.com/spatialos/UnrealGDKStarterProject
+* **REPOSITORY:** github.com/spatialos/UnrealGDKTestGyms
 
-This Starter Project is both a reference project and a starting point for building games on SpatialOS and Unreal Engine 4.  The project uses the `ThirdPersonExampleMap` and a controllable character `AStarterGameCharacter` on a SpatialOS deployment.
+This project is for testing the Unreal GDK. It contains a set of gyms as well as being a minimal project in which to run unit tests.
 
 For setup instructions, follow the [Get started](https://docs.improbable.io/unreal/latest/get-started/introduction) guide in the GDK for Unreal documentation.
```

Comment:
These set up instructions talk you through the Example Project and/or the Starter Template project but assume you use auto-install - you don't need to clone the Example Project repo.

There is also a manual install for tboth. I think we should suggest following the Example Project manual-install instructions, replacing links to the Example Project with links to the Test Gyms Project.


```suggestion
**Prerequisite:**

To use this project, you must have followed the [Get started](https://docs.improbable.io/unreal/latest/get-started/introduction) guide in the GDK for Unreal documentation and set up either of the Example Project or the Starter Template project.

**Set up**
* Follow the [manual-install - clone the project](https://docs.improbable.io/unreal/latest/content/get-started/example-project/exampleproject-manual-setup#step-2-clone-the-example-project-repository) instructions for the Example Project.
* Replace the Example Project repository link in the instructions with the Test Gyms Project repository link.
```